ThunderCloud/GridItem.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  GridItem.swift
+//  ThunderCloud
+//
+//  Created by Simon Mitchell on 16/02/2018.
+//  Copyright © 2018 threesidedcube. All rights reserved.
+//
+
+import UIKit
+import ThunderCollection
+
+/// A root class for an item to be displayed in a `grid` or `UICollectionView`
+open class GridItem: CollectionItemDisplayable, StormObjectProtocol {
+	
+	open var collectionCellClass: AnyClass? {
+		return UICollectionViewCell.self
+	}
+	
+	/// A `StormLink` which determines what the item does when it is selected
+	open let link: StormLink?
+	
+	/// The image to be displayed when displaying this item in a `UICollectionView`
+	open var image: UIImage?
+	
+	/// The title to be displayed when displaying this item in a `UICollectionView`
+	open let title: String?
+	
+	/// The description to be displayed when displaying this item in a `UICollectionView`
+	open let description: String?
+	
+	public required init?(dictionary: [AnyHashable : Any]) {
+		
+		remainSelected = false
+		
+		if let linkDicationary = dictionary["link"] as? [AnyHashable : Any] {
+			link = StormLink(dictionary: linkDicationary)
+		} else {
+			link = nil
+		}
+		
+		if let titleDict = dictionary["title"] as? [AnyHashable : Any] {
+			title = StormLanguageController.shared.string(for: titleDict)
+		} else {
+			title = nil
+		}
+		
+		if let subtitleDict = dictionary["description"] as? [AnyHashable : Any] {
+			description = StormLanguageController.shared.string(for: subtitleDict)
+		} else {
+			description = nil
+		}
+		
+		image = StormGenerator.image(fromJSON: dictionary["image"])
+	}
+	
+	public func configure(cell: UICollectionViewCell, at indexPath: IndexPath, in collectionViewController: CollectionViewController) {
+		
```

Comment:
Empty method? Remove if not needed

Reply:
This needs to be implemented otherwise for some reason the function doesn't get called on subclasses... No idea why!